src/model/saveScore.js

```diff
@@ -0,0 +1,29 @@
+export const saveScore = (gameType, userName, userScore) =>{
+
+    let scoreTable = getScores(gameType);
+
+    const scoreObject = {
+        'name': userName, 
+        'score': userScore
+    };
+
+    if(!scoreTable){
+        scoreTable = [];
+    }
+
+    scoreTable.push(scoreObject);
+
+    const userScore_serialized = JSON.stringify(scoreTable);
+
```

Comment:
I w całym tym pliku. Trochę nieładnie to wygląda.

Reply:
+1